app/experimenter/experiments/tests/api/v6/test_serializers.py

```diff
@@ -41,7 +41,7 @@ def test_serializer_outputs_expected_schema_with_feature(self):
                 # DRF manually replaces the isoformat suffix so we have to do the same
                 "endDate": experiment.end_date.isoformat().replace("+00:00", "Z"),
                 "id": experiment.slug,
-                "isEnrollmentPaused": False,
+                "isEnrollmentPaused": True,
```

Comment:
~Hmm, trying to remember why this flipped from False to True.~ Oh! I think I remember: It's because we made the change to factory-generated experiments to include pausing enrollment as part of the canned lifecycles.

Reply:
Right, maybe I shouldn't have resolved [this comment](https://github.com/mozilla/experimenter/pull/5402/files#r649338991) as it would have been more obvious. Guess this wasn't really necessary but 🤷‍♀️